pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -14,8 +14,8 @@ import (
 	"github.com/samalba/dockerclient"
```

Comment:
@ezrasilvera  In order to minimize changes to code not related to volume_binding it will be necessary to add an extra encode/decode of the request  to/from engine-api/types just for volume_binding.  Is this what you want?